src/pipx/commands/run.py

```diff
@@ -319,41 +326,45 @@ def _http_get_request(url: str) -> str:
         raise PipxError(str(e)) from e
 
 
+PEP723 = re.compile(
+    r"(?m)^# /// (?P<type>[a-zA-Z0-9-]+)$\s(?P<content>(^#(| .*)$\s)+)^# ///$"
+)
```

Comment:
A comment on the regex pattern (and/or maybe using `re.VERBOSE`) would be a good idea

Reply:
I'd say comment as long as we match the PEP's suggested regex.